codebuild/spec/buildspec_generalbatch.yml

```diff
@@ -16,7 +16,13 @@ version: 0.2
 
 batch:
   build-list:
-    - buildspec: codebuild/spec/buildspec_ubuntu.yml
+    - identifier: sawHMACPlus
+      # The HMAC SAW proof relied on the low-level "SHA256_CTX"-style hashing
+      # implementation. We updated to the modern EVP hashing implementation
+      # instead, breaking its assumptions about the memory layout.
+      # We either need to update the proof, or remove our custom HMAC implementation.
+      ignore-failure: true
```

Comment:
Is there any reason to continue running this test if it just always fails and we ignore the errors? Seems like kind of a waste. I wonder if this should just be documented in an issue or something instead?

Reply:
Not a good reason. Just a sad constant reminder of our choices :pensive: 

I'll just remove it and create an issue.